src/test/kotlin/no/nav/helse/AppComponentTest.kt

```diff
@@ -91,17 +95,17 @@ class AppComponentTest {
             "DATABASE_JDBC_URL" to embeddedPostgres.getJdbcUrl("postgres", "postgres")
         )) {
 
-            val søknadCounterBefore = getCounterValue("soknader_totals")
-            val søknadIgnorertCounterBefore = getCounterValue("soknader_ignorert_totals")
+            val søknadCounterBefore = getCounterValue(søknadCounterName)
+            val søknadIgnorertCounterBefore = getCounterValue(søknaderIgnorertCounterName)
 
-            val søknad = objectMapper.readTree("/søknad_om_utlandsopphold.json".readResource())
-            produceOneMessage("syfo-soknad-v2", søknad["id"].asText(), søknad)
+            val søknad = søknadObjectMapper.readTree("/søknad_om_utlandsopphold.json".readResource())
```

Comment:
liker ikke helt dette. Om vi ønsker å bruke objectmapperen i en test, da burde objectmapperen bli sendt inn i konstruktøren til SøknadConsumeren.

Reply:
Synes det er riktig at konsumenten eier objectmapperen, siden den har ansvar for å lese fra kafka. Så, når jeg tester, vil jeg være helt sikker på at konsumenten gjør det rette gitt korrekte data, og den enkleste måten å gjøre det på, er å låne objectmapperen derfra. Hvis testen eier objectmapperen som sendes inn til konsumenten, så får jeg ikke testet den koden som skal kjøre i prod like godt.